### ISSUE TITLE
<choose> inside <tee>

### ISSUE THREAD

**Issue by ahu-els** [cmarchand/gaulois-pipe#32]
Hello,

Is it possible to use a <choose> inside <tee>? The schema doesn't seem to allow this case :
```xml
  <pipe mutiThreadMaxSourceSize="24349456" nbThreads="4" traceOutput="#logger">
    <xslt href="cp:/common/xsl/identity.xsl"/>
    <tee>
      <pipe>
        <xslt href="cp:/common/xsl/identity.xsl"/>
        <choose></choose> // Not allowed
      </pipe>
      <pipe></pipe>
    </tee>
    <xslt href="cp:/common/xsl/identity.xsl"/>
    <output id="main" method="xml" encoding="UTF-8" indent="no">
      <folder absolute="$[CONTEXT.URI]/$[resStep]"/>
      <fileName name="$[input-basename].xml"/>
    </output>
  </pipe>
```
Can you check this point, to see you're having the same issue ?
I'm using version 1.01.11.

Thanks

**Comment by cmarchand** [comment]
That's correct, schema does not allow a `choose` in a tee.
But we should allow this.

**Comment by cmarchand** [comment]
Solved.
Will be available in 1.03.03.
Probably also available in 1.03.03-RC2
